client/src/featureform/tls.py

```diff
@@ -15,3 +19,12 @@ def secure_channel(host, cert_path):
         credentials = grpc.ssl_channel_credentials()
     channel = grpc.secure_channel(host, credentials)
     return channel
+
+
+def check_up_to_date(local, client):
```

Comment:
probably the wrong python package for this? Not a huge deal though will leave it up to you

Reply:
lemme see if i can find a better place